packages/lesswrong/components/posts/PostsPage/ContentType.tsx

```diff
@@ -97,6 +99,14 @@ export const contentTypes: ForumOptions<ContentTypeRecord> = {
       linkTarget: curatedUrl,
       Icon: StarIcon,
     },
+    event: {
```

Comment:
Is this related to the rest of the PR?

Reply:
Yes, when you do the header redesign, Event posts look weirdly orphaned/lonely at the top-right because they don't have tags, and the design kinda depended on having at least one tag-ish-thing to look reasonable.